apps/emqx_bridge/src/schema/emqx_bridge_schema.erl

```diff
@@ -41,12 +41,16 @@ api_schema(Method) ->
     hoconsc:union(Broker ++ EE).
 
 ee_api_schemas(Method) ->
+    %% must ensure the app is loaded before checking if fn is defined.
+    ensure_loaded(emqx_ee_bridge, emqx_ee_bridge),
```

Comment:
I guess this only happens in tests, maybe move the loading to test helper module.
so we do not expose yet another part of ee to ce

Reply:
I actually saw this problem in the build:

```
2022-09-02T09:38:23.464800-03:00 [error] failed_to_check_schema: emqx_ee_conf_schema
2022-09-02T09:38:23.465549-03:00 [error] #{kind => validation_error,path => "bridges",reason => unknown_fields,unknown => <<"mongodb_rs">>,unmatched => <<"mqtt,webhook">>}
ERROR: call_hocon_failed: -v -t 2022.09.02.09.38.23 -s emqx_ee_conf_schema -c /home/thales/dev/emqx/emqx/_build/emqx-enterprise/rel/emqx/etc/emqx.conf -d /home/thales/dev/emqx/emqx/_build/emqx-enterprise/rel/emqx/data/configs generate
```